.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release to npm and gh-pages.
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: A job to execute release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install packages.
+        run: yarn install
+      - name: Set up project.
+        run: yarn build
```

Comment:
Shouldn't we use `yarn bootstrap` here? That seems to do more checks and other stuff like builing types

Reply:
We can. It seems to work with build as well and it is quicker, but we can switch to bootstrap, no problem if you think it is meaningful

Reply:
It seems meaningful, switched to bootstrap